markdown/events/2023/hackathon-march-2023/index.md

```diff
@@ -44,22 +56,64 @@ Watch this space for more details in the coming weeks!
 | 🇺🇸 USA          | San Jose   | Quilt Co-Working Space                    | [Read more](usa-san-jose.md)              |
 | 🇺🇸 USA          | Wyoming    | The University of Wyoming                 | [Read more](usa-university-wyoming.md)    |
 
-Please view the event page for a specific location if you're interested in attending.
+Please view the event page above for location-specific information. Also, if you intend to attend a local site, make sure you select it during your registration so the local host knows you will be attending.
 
-If there is no event nearby, do not worry - everyone is welcome to join online!
+If there is no event near you, don't worry - everyone is welcome to join online!
 
 ## Hosting an event
 
-Would you be interested in hosting a local event? All you need to do is book a room and tell some people!
-Please put in a pull request to edit this file to add the details of your event so that others can find you.
+Hosting a local event can be as simple as booking a room and telling your friends!
+
+If you are interested in adding a local site, please put in a pull request to edit this page and add the details of your event so that others can find you. Also, let the nf-core outreach team know as soon as possible so we can add your site to the registration form, mail you some goodies, and help you with the organization.
+
+## Groups
+
+To keep things manageable everyone will be organized into groups.
+You are free to change groups at any time during the hackathon.
+
+We will coordinate our work and the issues we are working on using a single GitHub project board (we'll post a link here closer to the time).
+There will be a separate tab for each group where will have a list of relevant issues for you to work on.
+
+#### Documentation
+
+The documentation group will focus on all things documentation related, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. This is a great group to start, if you are not sure where to begin.
+
+#### Infrastructure
+
+The nf-core website and central tooling are always being improved. If you enjoy working with python or want to do some web development (mainly CSS and javascript (we are using astrojs+svelte)), join the infrastructure group!
+
+#### Pipelines
+
+The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)).
```

Comment:
```suggestion
The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)). These are typically lead by the main pipeline developers,  if they also attending the hackathon.
```

Or something?